docs/install_guides/installing-openvino-docker-windows.md

```diff
@@ -1,78 +1,98 @@
-# Install Intel® Distribution of OpenVINO™ toolkit for Windows* from Docker* Image {#openvino_docs_install_guides_installing_openvino_docker_windows}
+# Install Intel® Distribution of OpenVINO™ toolkit for Windows from Docker Image {#openvino_docs_install_guides_installing_openvino_docker_windows}
 
-The Intel® Distribution of OpenVINO™ toolkit quickly deploys applications and solutions that emulate human vision. Based on Convolutional Neural Networks (CNN), the toolkit extends computer vision (CV) workloads across Intel® hardware, maximizing performance. The Intel® Distribution of OpenVINO™ toolkit includes the Intel® Deep Learning Deployment Toolkit.  
+This guide provides steps for creating a Docker image with Intel® Distribution of OpenVINO™ toolkit for Windows* and using the Docker image on different devices.
 
-This guide provides device specifics for a Docker* image creation with Intel® Distribution of OpenVINO™ toolkit for Windows* and its further usage.
+There are two options to install OpenVINO with Docker:
 
-## System Requirements
+* You can <a href="#using-prebuilt-image">get a prebilt image and run the image directly</a>;
+* Or <a href="#build-image-manually">build a docker image manually and run the image on different devices</a>.
 
-**Target Operating Systems**
 
-- Windows Server Core*
+## System requirements
 
-**Host Operating Systems**
+@sphinxdirective
+.. tab:: Target Operating Systems
 
-- Windows 10*, 64-bit Pro, Enterprise or Education (1607 Anniversary Update, Build 14393 or later) editions
-- Windows Server* 2016 or higher
+  Windows Server Core
```

Comment:
Please add the following to the target operating systems:
* Windows Server Core* OS
* Windows base* OS

Reply:
updated